src/js/_enqueues/wp/theme.js

```diff
@@ -1694,7 +1694,18 @@ themes.view.Installer = themes.view.Appearance.extend({
 	browse: function( section ) {
 		// Create a new collection with the proper theme data
 		// for each section.
-		this.collection.query( { browse: section } );
+		if ( 'block-themes' === sort ) {
```

Comment:
It needs to be `blockthemes`, same as the `data-sort` attribute in the `theme-install.php` file.

Also, the variable to check against needs to be `section` not `sort`. Then it works for me.

Reply:
Ah yeah I messed up in my last commit… thanks 👍